### PR TITLE
Use foreman

### PR DESCRIPTION
**Pull request by pocke**
- [x] Install foreman
- [x] Enable to work pry when using foreman
- [x] Replace command under scripts/
  - [x] dev_server.sh
  - [x] skyhopper_daemon.sh
- [x] Improve message of SkyHopper status in scripts/skyhopper_daemon.sh

### CODE REVIEW

**Comment by pocke** [comment]
done

please review and merge it @root-beer

**Comment by pocke** [comment]
`pry-remote` is required if we use foreman. 

see: https://skyarch.backlog.jp/view/NC-504

**Comment by Joeper214** [comment]
can I also review? @pocke-san? but @root-beer will merge.

**Comment by pocke** [comment]
@Joeper214 Thanks! please

**Comment by Joeper214** [comment]
@pocke . Do you know this error? 

xe_error

**Comment by pocke** [comment]
@Joeper214 No... I think cause of this error is unexpected request.

**Comment by pocke** [comment]
If port 80 of SkyHopper instance is open to external, maybe this is an attack.

**Comment by Joeper214** [comment]
okay. but your instance does not have this?

**Comment by pocke** [comment]
My instance doesn't have this. because I use SkyHopper on local PC.

**Comment by Joeper214** [comment]
okay. Thanks!. I closed external for port 80.

File: scripts/skyhopper_daemon.sh [hunk 1]
```diff
@@ -17,16 +17,27 @@ fi
...
+pid_file() {
+  pid_path="${SKYHOPPER_PATH}/tmp/pids/foreman.pid"
+  echo -n $pid_path
+}
 
 get_pid() {
-  pid_path="${SKYHOPPER_PATH}/tmp/pids/unicorn.pid"
+  pid_path=$(pid_file)
   if [ -f "${pid_path}" ]; then
-    cat $SKYHOPPER_PATH/tmp/pids/unicorn.pid
```
Comment (Joeper214) [inline scripts/skyhopper_daemon.sh]: Do you still use unicorn?

Comment (pocke) [inline scripts/skyhopper_daemon.sh]: Yes, foreman starts rails(unicorn)/sidekiq/websocket servers.

Comment (Joeper214) [inline scripts/skyhopper_daemon.sh]: :+1:

**Comment by Joeper214** [comment]
LGTM!

**Pull request merged**